### DISCUSSION TITLE
USB CDC can work, but tud_mount_cb tud_umount_cb callback function keeps triggering

### DISCUSSION THREAD

**Discussion by arilink-tech** [hathach/tinyusb#998] [Q&A]
I transplanted RT-Thread and TinyUSB on STM32F105. I found a problem during the test. When I output the status through printf in the tud_mount_cb() and tud_umount_cb() callback functions, I found that these two callback functions have been triggered, and the USB cannot be normal. Work When I use printf to output status, USB can work normally, CDC can send and receive normally. Why?

**Comment by hathach** [comment]
I cannot tell, since there is little info, it is probably your integration issue.
